test/appengine/test_urlfetch.py

```diff
@@ -0,0 +1,51 @@
+from . import MockResponse, activate_sandbox
+activate_sandbox()
```

Comment:
Hmm, if that's the case then we could have it only import this and not-skip tests iff some environment variable is present.